### PR TITLE
[Improve][Zeta][Config] change Zeta config

### PR DESCRIPTION
**Pull request by liugddx**
## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/incubator-seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/incubator-seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/incubator-seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by liugddx** [comment]
#4044

**Comment by liugddx** [comment]
This configuration can isolate the rest api.

File: config/hazelcast-client.yaml [hunk 1]
```diff
@@ -21,4 +21,4 @@ hazelcast-client:
     hazelcast.logging.type: log4j2
   network:
     cluster-members:
-      - localhost:5801
\ No newline at end of file
+      - localhost:9000
```
Comment (EricJoy2048) [inline config/hazelcast-client.yaml]: Why change the port?

**Review by EricJoy2048 — COMMENTED** [review]
I suggest enable the zeta e2e test to test this pr.

File: config/hazelcast-client.yaml (hunk 1 above)
Comment (liugddx) [reply config/hazelcast-client.yaml]: > Why change the port?

refer to https://docs.hazelcast.com/imdg/4.2/clusters/advanced-network-configuration#latest-banner

Each endpoint configuration that defines a server socket must bind to a different socket address.

Comment (EricJoy2048) [reply config/hazelcast-client.yaml]: > > Why change the port?
> 
> refer to https://docs.hazelcast.com/imdg/4.2/clusters/advanced-network-configuration#latest-banner
> 
> Each endpoint configuration that defines a server socket must bind to a different socket address.

Ok, Can you update the document of zeta cluster deployment?

File: config/hazelcast-client.yaml [hunk 2]
```diff
@@ -21,4 +21,4 @@ hazelcast-client:
     hazelcast.logging.type: log4j2
   network:
     cluster-members:
-      - localhost:5801
```
Comment (Hisoka-X) [inline config/hazelcast-client.yaml:24]: Hi, Why change port to 9000?

File: config/hazelcast-client.yaml (hunk 1 above)
Comment (liugddx) [reply config/hazelcast-client.yaml]: @Hisoka-X

File: config/hazelcast-client.yaml (hunk 2 above)
Comment (EricJoy2048) [reply config/hazelcast-client.yaml:24]: > Hi, Why change port to 9000?

Can we keep using port 5801? Some of our users are already using seatunnel cluster, and modifying the port will have some impact.

Comment (liugddx) [reply config/hazelcast-client.yaml:24]: ok

**Review by EricJoy2048 — APPROVED** [review]

**Review by Hisoka-X — APPROVED** [review]

**Pull request merged**